src/main/java/Entities/Address.java

```diff
@@ -0,0 +1,64 @@
+package Entities;
+

```

Comment:
I think you don't need two blank lines here.

Reply:
Removed it!!